### PR TITLE
#26 broke the build- fix histogram tests and keep compatibility with older golang versions

### PR DESCRIPTION
**Pull request by bejelith**
- Build broke by #26 so had to change go.mod.
- Reworked tests to make more resilient in case of dependency changes.

### CODE REVIEW

**Comment by bejelith** [comment]
is there anybody working on this repository ? @joannak-vmware @vikramraman @laullon

**Comment by joannak-vmware** [comment]
Hi @bejelith , I added myself as reviewer since I did the version bump, and will add an additional reviewer soon

File: go.mod [hunk 1]
```diff
@@ -3,8 +3,7 @@ module github.com/wavefronthq/go-metrics-wavefront
 go 1.13
 
 require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.5.1
```
Comment (keep94) [inline go.mod:7]: Will you please explain why you can't use the latest version of testify?

Comment (bejelith) [reply go.mod:7]: Their travis buils against very old versions of golang that do not support 1.7. the question is why are they still testing against golang 1.13?

Comment (keep94) [reply go.mod:7]: ok thanks for fixing this.

**Review by keep94 — APPROVED** [review]

**Review by joannak-vmware — APPROVED** [review]

**Comment by bejelith** [comment]
I cannot merge it as im not authorized

**Comment by joannak-vmware** [comment]
@bejelith , have merged changes.

**Pull request merged**